src/commands/SinceCommand.ts

```diff
@@ -213,7 +213,8 @@ async function execSinceCommandAux(destinationRoomId: string, event: any, mjolni
             } else if (maybeRoom.startsWith("#") || maybeRoom.startsWith("!")) {
                 const roomId = await mjolnir.client.resolveRoom(maybeRoom);
                 if (!protectedRooms.has(roomId)) {
-                    return mjolnir.managementRoomOutput.logMessage(LogLevel.WARN, "SinceCommand", `This room is not protected: ${htmlEscape(roomId)}.`);
```

Comment:
This was found as a result of the any return type of logMessage. I fixed it with what I assume may have been intended here. But logMessage can only return void. So the return made no selve (and now causes a typescript error as it should)